### PR TITLE
Models created.

### PR DESCRIPTION
**Pull request by COHally**
## Describe your changes
Models created for Django back end. 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## What kind of feedback do you need?
Models/database schema have already been discussed at length to a quick look will be sufficient. 
## Checklist:
- [x] I have performed a self-review of my code
- [x] I have recommended where to start and how to proceed with the review

### CODE REVIEW

**Review by caoimhejyoti — APPROVED** [review]
Looking brilliant. Nice work getting this smashed out!

File: byte_time/events/models.py
Comment (caoimhejyoti) [inline byte_time/events/models.py:1]: Damn! how did we miss this!

File: byte_time/events/views.py
Comment (caoimhejyoti) [inline byte_time/events/views.py:1]: I don't think that the createEvent class is correct. Referring to my previous back end, I had a Festival list which included a get an post. I then had a detailed list which had the get, put and delete.

Comment (caoimhejyoti) [inline byte_time/events/views.py:1]: Your delete references pledge - possibly pulled from previous project?

Also the comment "#Update Projects" on line 49 is possible the same.

**Review by caoimhejyoti — CHANGES_REQUESTED** [review]
Looking great Cathy - just a couple of things that I have spotted.

File: byte_time/events/models.py
Comment (caoimhejyoti) [inline byte_time/events/models.py:1]: There is a possible loop between the event mentor and the event table - can we just talk that through this possible loop and confirm?

**Review by caoimhejyoti — CHANGES_REQUESTED** [review]
see comment re:event mentor/mentor table loop.

File: byte_time/events/models.py
Comment (COHally) [reply byte_time/events/models.py:1]: Yes, of course.

**Review by caoimhejyoti — APPROVED** [review]
NAILED IT!

**Review by Jigudhari — APPROVED** [review]
Great Job Cathy!!

**Pull request merged**